README.md

```diff
@@ -2,6 +2,13 @@
 
 Multi-user application for tracking the educational process.
 
+### Technologies
+
+- React
+- TypeScript
+- Vite
+- SCSS modules
+
```

Comment:
Maybe add Redux. We planned to use it

Reply:
But we don't use it now. We can add Redux when will use it)

Reply:
Ok